.github/bin/s3/delete-s3-bucket.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
```

Comment:
why no `-e`?

Reply:
I assume this is because we don't want this script to make the CI run fail, but this should be handled in the CI step (ignoring the exit code).

Reply:
Correct

Reply:
How would I ignore the exit code?

Reply:
I changed this command to `.github/bin/s3/delete-s3-bucket.sh || true`, I think that should ignore the exit code

Reply:
and set `-e`

Reply:
thanks